src/app/util/types_stub.h

```diff
@@ -1994,6 +1994,8 @@ typedef struct
  */
 #define halCommonSetIndexedToken(token, index, data)
 
+#define COUNTOF(a) (sizeof(a) / sizeof(a[0]))
```

Comment:
CHIP exposes `ArraySize()` in src/lib/support/CodeUtils.h, recommend we use that instead

Reply:
Sounds good. I will change the code.